msdm/tests/test_core_distributions.py

```diff
@@ -1,9 +1,8 @@
 import unittest
 import numpy as np
-
 import pandas as pd
-
-from msdm.core.distributions import DiscreteFactorTable as Pr
+from scipy.special import softmax
```

Comment:
According to [Blanchard et al. (2020)](https://academic.oup.com/imajna/advance-article/doi/10.1093/imanum/draa038/5893596), scipy uses a division-free version of the softmax calculation, which is not recommend as it can lead to a loss in floating point accuracy. Instead we should probably just calculate it ourselves.

Reply:
This is a bigger concern that we should deal with in a separate issue (see https://github.com/markkho/msdm/issues/12#issue-836357378)